embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/capture/envelope/resource/EnvelopeResourceSourceImpl.kt

```diff
@@ -1,9 +1,60 @@
 package io.embrace.android.embracesdk.capture.envelope.resource
 
+import io.embrace.android.embracesdk.Embrace.AppFramework
 import io.embrace.android.embracesdk.internal.payload.EnvelopeResource
+import io.embrace.android.embracesdk.payload.AppInfo
+import io.embrace.android.embracesdk.payload.DeviceInfo
 
-internal class EnvelopeResourceSourceImpl : EnvelopeResourceSource {
+internal class EnvelopeResourceSourceImpl(
+    private val deviceInfo: DeviceInfo,
+    private val appInfo: AppInfo,
```

Comment:
Would be ideal to avoid these types as they're the old payload that will eventually be deleted. Is this addressed in another changeset?